packages/styles/src/components/button.scss

```diff
@@ -105,8 +108,11 @@ tokens.$default-map: components.$post-button;
   }
 }
 
-// Tertiary with no padding (overrides the padding defined by the sizing classes above)
 .btn-link {
-  padding-inline-start: 0;
-  padding-inline-end: 0;
+  text-decoration: underline;
+  min-height: 0;
+  font-size: inherit;
+  font-weight: inherit;
+  padding: 0;
+  border-radius: tokens.get('focus-border-radius', helpers.$post-focus);
 }
```

Comment:
Should we use the link tokens here (which are defined in the file `elements/anchor.scss`)?
What do you think?

Reply:
yes, good idea! I used the ones defined there for text-decoration, radius and hover color :)